core/controllers/dashboard_test.py

```diff
@@ -14,7 +14,7 @@
 
 """Tests for the user notification dashboard and 'my explorations' pages."""
```

Comment:
Nit: please update docstring to reflect these tests are for the creator dashboard.